runtime-interface/src/lib.rs

```diff
@@ -70,14 +74,85 @@ impl PassBy for LazyPagesProgramContext {
 
 #[derive(Debug, Clone, Encode, Decode)]
 #[codec(crate = codec)]
-pub struct LazyPagesRuntimeContext {
+pub struct LazyPagesInitContext {
```

Comment:
whats the point of renaming?

Reply:
It better conveys meaning of struct

Reply:
Runtime context means context which is constant for current runtime. Although I am not against this  renaming